R/i18n.R

```diff
@@ -128,6 +128,18 @@ i18n_span <- function(key, ..., opts = NULL) {
   htmltools::HTML(format(x))
 }
 
+i18n_combine_words <- function(
+  words, and = c("and", "or"), before = "", after = before
+) {
+  and <- match.arg(and)
+  and <- paste0(" $t(text.", and, ") ")
```

Comment:
Very much a nit, but we can use old-school glue here.

```suggestion
  and <- sprintf(" $t(text.%s) ", and)
```